code/modules/reagents/machinery/dispenser/cartridge.dm

```diff
@@ -55,10 +55,10 @@
 	..()
 	if (is_open_container())
 		to_chat(usr, "<span class = 'notice'>You put the cap on \the [src].</span>")
-		flags ^= OPENCONTAINER
+		atom_flags ^= ATOM_REAGENTS_IS_OPEN
 	else
 		to_chat(usr, "<span class = 'notice'>You take the cap off \the [src].</span>")
-		flags |= OPENCONTAINER
+		atom_flags |= ATOM_REAGENTS_IS_OPEN
```

Comment:
This pains me

Reply:
And there's more to come!

Just, easier, since just rename-in-place.

Reply:
Moreso the `if(condition) print(text) else print(basically the same text)`